docs/sdks/server-sdks/node.md

```diff
@@ -325,15 +325,15 @@ It may take up to 10 seconds for changes to Eppo experiments to be reflected by
 
 ## Typed assignments
 
-Use the following getAssignment method for the type of feature flag created:
+Use the following `get<Type>Assignment` methods for the type of feature flag created:
 
 ```
 getBoolAssignment(...)
 getNumericAssignment(...)
 getJSONStringAssignment(...)
 getParsedJSONAssignment(...)
```

Comment:
These are also changing, there's going to be only a `getJSONAssignment`

Reply:
Ack - I didn't want to include any changes from v3 itself in this PR. Just what has already changed (the deprecated function call, for example), to get ahead on the v3 changes proper.